src/components/Auth/Auth.js

```diff
@@ -22,6 +22,15 @@ class Auth extends PureComponent {
 
   static defaultProps = defaultProps
 
+  constructor(props) {
+    super()
```

Comment:
you don't need to call `super` here :P

Reply:
@jarindr it's required

Reply:
it breaks if we remove it ? hmmm...

Reply:
babel throws `ReferenceError: this hasn't been initialised - super() hasn't been called`
probably assertion that `this` will be available for other methods, even if it's not needed in constructor

Reply:
okay the, thanks for fixing :D !

Reply:
anytime, thanks for review :)